Makefile

```diff
@@ -11,15 +11,28 @@ create_project:
 # Y - in response to Do you want to generate the .github/ISSUE_TEMPLATE and .github/PULL_REQUEST_TEMPLATE?
 # Y - in response to Do you want to generate the Github Action workflow?
 # Y - in response to Would you like to add the Oban addon?
+common_addon_prompts = Y\nY\nY\nY\n
+
+# Y - in response to Would you like to add the CoreJS addon?
+web_addon_prompts = Y\n
+
+api_addon_prompts = 
+
 # Y - in response to Fetch and install dependencies?
+suffix_addon_prompts = Y\n
```

Comment:
`suffix` is kinf of awkward here 🤔 How about `post_setup_addon_prompts. This would be similar to [NPM](https://docs.npmjs.com/cli/v6/using-npm/scripts):

![Screen Shot 2563-11-26 at 16 59 36](https://user-images.githubusercontent.com/696529/100336514-dc5ebd00-3008-11eb-8ef8-3b19d6ac0540.png)

Reply:
Fixed in https://github.com/nimblehq/elixir-templates/pull/58/commits/fa6253fd3a29280ed91c3b6d0d457e075ef04df4